stackslib/src/clarity_vm/tests/events.rs

```diff
@@ -288,6 +289,24 @@ fn test_emit_nested_print_nok() {
     assert_eq!(events.len(), 1);
 }
 
+#[cfg(feature = "clarity-wasm")]
```

Comment:
I'm actually unsure we should keep this flag here. The test is supposed to work both with the interpreter and wasm.
This would mean we should change the name of the test too, to something like `test_emit_nested_print_nok_valid_contract` or something like that.